src/Components/ScrivitoExtensions/ColumnsEditorTab.js

```diff
@@ -185,6 +191,13 @@ const Alignment = Scrivito.connect(({ widget }) => {
       break;
   }
 
+  if (Scrivito.canWrite()) {
+    startAlignmentClasses.push("clickable");
+    centerAlignmentClasses.push("clickable");
+    endAlignmentClasses.push("clickable");
+    stretchAlignmentClasses.push("clickable");
```

Comment:
* [ ] FYI the four initial values are all equal. We could refactor to:
  * Calculate conditional initial value once
  * Assign initial value four times